resources/assets/lib/quick-search/worker.ts

```diff
@@ -42,14 +55,77 @@ interface SearchResultUser extends SearchResultSummary {
   users: User[];
 }
 
+const otherModes: ResultMode[] = ['forum_post', 'wiki_page'];
+
 export default class Worker {
+  @observable cursor: Cursor | null = null;
```

Comment:
`cursor` and `Cursor` are used to refer to the api response's `cursor`, name it something else